spacemouse-keys/config_sample.h

```diff
@@ -238,3 +239,35 @@
 #endif
 
 #define DEBOUNCE_KEYS_MS 200  // time in ms which is needed to allow a new button press
+
+// -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+// ENCODER WHEEL
+// Needs the encoder library by Paul Stoffregen.
+// There may be an additional encoder wheel to replace one of the velocities
+#define SIMAXIS 0 
```

Comment:
I think that the name of this variable is not understandable. What is the link between the zoom functionality and "sim" ?

Reply:
Yes, good point! 

I suggest ENCODERVELOCITY, because the Encoder replaces one velocity (and not an "axis").

Agreed or do you have an other suggestion?

Reply:
As a non english speaker, velocity means for me something like "speed". As you wrote line 249, this option is intended to replace an axis even if you say that is a velocity or a movement. I would suggest `ROTARY_AXIS`.

Reply:
Updated in 39b227a

Reply:
Perfect !